libpcsxcore/cdrom.h

```diff
@@ -83,6 +83,7 @@ typedef struct {
 	unsigned char SetSector[4];
 	unsigned char Track;
 	boolean Play, Muted;
+	boolean m_locationChanged;
 	int CurTrack;
```

Comment:
This will trash savestate compatibility because of cdr struct size change. You can replace one of pad* (padding) members with this instead.

I used to have tons of savestates to test problematic things in games. Who knows if I'll ever use them again, but you can easily avoid breaking them.